### DISCUSSION TITLE
Action wheel is still empty

### DISCUSSION THREAD

**Discussion by celinathemoon** [ErinaSugino/Starbound-SBR-Mod#100] [Trouble Shooting]
i searched almost 5 hours but there is no solution about this situlation
i only downloaded main mod and gof###yourself but action wheel is still empty how do i fix it?

**Comment by ErinaSugino** [comment]
We will need a log file to be able to tell what going on, or wrong.

In order to obtain a clean, helpful log, first edit the main `sexbound.config` file and, near the bottom of that file, set the setting `showDebug` and `showInfo` to `true`. Then reproduce the issue you are having after a clean start of the game. Try to do as little as possible besides triggering the issue, as to not overload the log file with too much unnecessary stuff. Finally, navigate to the `storage` folder in your Starbound installing directory, and grab the **plain text file** `starbound.log`. If you have file endings disabled, it's just called `starbound`. Don't use any of the log files that have a number at the end of them, like "log.1" - these are old backup logs the game keeps.

Then just drag and drop the file here to attach it.
